core/metacling/src/TCling.cxx

```diff
@@ -2824,6 +2824,17 @@ Bool_t TCling::IsLibraryLoaded(const char* libname) const
    return s_IsLibraryLoaded(libname, fInterpreter);
 }
 
+Bool_t TCling::HasPCMForLibrary(const char *libname) const
+{
+   llvm::StringRef ModuleName(libname);
+   ModuleName = llvm::sys::path::stem(ModuleName);
+   ModuleName.consume_front("lib");
+
+   clang::ModuleMap &moduleMap = fInterpreter->getCI()->getPreprocessor().getHeaderSearchInfo().getModuleMap();
+   clang::Module *M = moduleMap.findModule(ModuleName);
+   return M && !M->IsMissingRequirement;
```

Comment:
Could you change this to `M && m->getASTFile() && !M->IsMissingRequirement`.

Reply:
Oh wow, this feature is super cool